src/components/command-bar/commands/search/unified-search/components/unified-hits-container/index.tsx

```diff
@@ -64,14 +64,14 @@ export function UnifiedHitsContainer({
 									</div>
 								</>
 							) : (
-								<>
+								<div className={s.noResultsWrapper}>
```

Comment:
nit: what do you think about `NoResultsMessage` accepting `className` and passing the class down to its root element? That way we avoid an additional wrapping element here.

Reply:
I can definitely see the case! Initially I reached for the wrapper cause 1) i have come to hate `margin` and sometimes just wanna do all spacing with `padding` (😭) and 2) `className` feels like it makes it easier to reach for "CSS override" type stuff... Then realized maybe that's something we can avoid through practice rather blocking it off completely 👍

So I went in to try to revise this to remove the extra `<div />`, but had trouble:
- There's a [CSS rule in Tabs that nukes `margin-top` on `:first-child`](https://github.com/hashicorp/dev-portal/blob/zs.unified-search-revise-empty-state/src/components/tabs/tabs.module.css#L80) with a `!important` 😬 (cruft from pre-MDX-component days I think?)
- Setting `padding` on `noResultsWrapper` as a separate wrapper works, but setting it on the same root element as `NoResultsMessage` means we undo the component's desired internal `padding`

With that in mind, left it as-is for now, and made a task 🎟 [Remove !important margin-top setting on Tabs tabpanel :first-child](https://app.asana.com/0/1202097197789424/1204918499533960/f) for us to fix up the root issue here in the future.

Reply:
Dang 🥲 well thanks for trying! 🙏